src/main/java/com/mybookfinder/enterprise/BookFinderController.java

```diff
@@ -115,8 +124,8 @@ public Book createBook(@RequestBody Book book)
     /**
      * Endpoint to delete book by ID
      */
-@DeleteMapping("/book/{title}")
-    public ResponseEntity deleteBook(@PathVariable("title") String title)
+@DeleteMapping("/book/{id}/")
+    public ResponseEntity deleteBook(@PathVariable("id") String id)
```

Comment:
Changed the endpoint to match what the comment describes it as. It would be better to delete books based off a unique Id because books can have the same titles.  As far as I can tell there isn't a standard practice for or against the trailing slash for URIs (https://stackoverflow.com/questions/61547014/restful-uri-trailing-slash-or-no-trailing-slash). I added one to make the URI consistent with the other URIs.
Source: Module 3 video: Create GET, POST, and DELETE endpoints in SpringBoot with IntelliJ